index.html

```diff
@@ -54,59 +49,80 @@
 The <dfn>`Promise`</dfn> and <dfn>`DOMException`</dfn> interfaces are defined
 in [[!DOM]].
 
-## <dfn>SerialPort</dfn> interface
+<section data-dfn-for="Serial" data-link-for="Serial">
+<h2><dfn>Serial</dfn> interface</h2>
+
 <pre class="idl">
-[constructor(DOMString path, optional SerialOptions options)]
+[Exposed=Window, SecureContext]
+partial interface Navigator {
+  readonly attribute Serial serial;
+};
+
+[Exposed=DedicatedWorker, SecureContext]
+partial interface WorkerNavigator {
+  readonly attribute Serial serial;
+};
+
+[Exposed=(DedicatedWorker,Window), SecureContext]
+interface Serial : EventTarget {
+  attribute EventHandler onconnect;
+  attribute EventHandler ondisconnect;
+  Promise&lt;sequence&lt;SerialPort>> getPorts();
+  [Exposed=Window] Promise&lt;SerialPort> requestPort(SerialPortRequestOptions options);
+};
+</pre>
+
+<section>
+<h2><dfn>requestPort()</dfn> method</h2>
+
+The <a>requestPort()</a> method requests access to a serial port of the user's
+choice (see <a>security considerations</a>).
+
+When the <a>requestPort()</a> method is invoked, the user agent MUST perform the
+following steps:
+
+ 1. Let <var>promise</var> be a new <a>`Promise`</a> object and
+    <var>resolver</var> its associated resolver.
+ 1. Return `promise` and run the remaining steps asynchronously.
+ 1. Request access to a serial port (see <a>security considerations</a>):
+   1. If the access request is rejected:
+     1. Let <var>error</var> be a new <a>`DOMException`</a> whose name is
+        `[AbortError](http://dom.spec.whatwg.org/#aborterror)`.
+     1. Run resolver's internal reject algorithm with <var>error</var> as value
+        and terminate this algorithm.
+   1. Otherwise, let <var>port</var> be the <a>`SerialPort`</a> for which access
```

Comment:
I'd like to leave that to a later patch because I actually want the values returned from `getPorts()` and `requestPort()` as well as the objects associated with (dis)connect events to have `===` equality which means defining an internal concept of a serial port to which a `SerialPort` instance can be assigned and used consistently thereafter.